drivers/clickhouse/clickhouse_test.go

```diff
@@ -0,0 +1,183 @@
+package clickhouse_test
+
+import (
+	"database/sql"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"testing"
+
+	dt "github.com/ory/dockertest/v3"
+	dc "github.com/ory/dockertest/v3/docker"
+	"github.com/xo/usql/drivers/clickhouse"
+	"github.com/xo/usql/drivers/metadata"
+)
+
+type Database struct {
+	BuildArgs  []dc.BuildArg
+	RunOptions *dt.RunOptions
+	Exec       []string
+	Driver     string
+	URL        string
+	DockerPort string
+	Resource   *dt.Resource
+	DB         *sql.DB
+	Opts       []metadata.ReaderOption
+	Reader     metadata.BasicReader
+}
+
+var dbName string = "clickhouse"
+
+var db = Database{
+	BuildArgs: []dc.BuildArg{
+		{Name: "BASE_IMAGE", Value: "yandex/clickhouse-server:21.7.4.18"},
+		{Name: "SCHEMA_URL", Value: "https://raw.githubusercontent.com/jgryko5/usql/clickhouse_driver/drivers/clickhouse/testdata/clickhouse.sql"},
```

Comment:
`SCHEMA_URL` needs to be changed once the file is merged to the master branch.

Reply:
@jgryko5 when I reviewed this before, I was going to suggest changing just copying the file locally into the container. Not really sure what retrieving it from a remote URL would accomplish.